content/errors.xql

```diff
@@ -32,6 +32,10 @@ declare variable $errors:UNAUTHORIZED := xs:QName("errors:UNAUTHORIZED_401");
 declare variable $errors:FORBIDDEN := xs:QName("errors:FORBIDDEN_403");
 declare variable $errors:NOT_FOUND := xs:QName("errors:NOT_FOUND_404");
 declare variable $errors:METHOD_NOT_ALLOWED := xs:QName("errors:METHOD_NOT_ALLOWED_405");
+(: https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.13 :)
+declare variable $errors:UNSUPPORTED_MEDIA_TYPE := xs:QName("UNSUPPORTED_MEDIA_TYPE_415");
```

Comment:
The QName should be `"errors:UNSUPPORTED_MEDIA_TYPE_415"` to put it into the same namespace as the others.